### DISCUSSION TITLE
posts/5-github-actions-to-automate-your-react-native-workflow

### DISCUSSION THREAD

**Discussion by giscus** [yjose/elazizi.com#112] [Blog posts]
# posts/5-github-actions-to-automate-your-react-native-workflow

In this article, I will share with you 7+ Github Actions to Automate Your React Native Workflow.

https://elazizi.com/posts/5-github-actions-to-automate-your-react-native-workflow

**Comment by pashtet86** [comment]
Hi, thank you, the article is great and the info is useful.
